scripts/setup

```diff
@@ -5,16 +5,23 @@ for tool in "${@}"; do
   echo "Installing '$tool'..."
   export tool
   install_method="$(jq -r '.[env.tool].method' dev-tools.json)"
+  target="$HOME/.local/bin/$tool"
+  export target
   echo "  install_method: $install_method"
   case "$install_method" in
+  "apt")
+    sudo apt-get install -yy "${tool}"
+    ;;
   "curl")
-    url="$(jq -r '.[env.tool].url' dev-tools.json)"
-    target="$HOME/.local/bin/$tool"
+    url="$(jq -r '.[env.tool].version as $version | .[env.tool].url | gsub("[$]{VERSION}"; $version)' dev-tools.json)"
+    pipe=($(jq -r '.[env.tool].pipe // "cat"' dev-tools.json))
     mkdir -p "$(dirname "$target")"
-    curl -Lf "$url" | install -m 755 /dev/stdin "$target"
+    curl -Lf "$url" | "${pipe[@]}" | install -m 755 /dev/stdin "$target"
```

Comment:
This allows us to pipe through commands with flags.  Previously only say `gunzip` was used, but we will need `tar -xz --to-stdout` which , with the old code, would fail as there is no executable called `tar -xz --to-stdout` (all one word).